### PR TITLE
Document the label group e #4756

### LINKED ISSUE #4756: Document the label group e
e.1, e.2, e.4, etc.

### PR DESCRIPTION
**Pull request by damithc**
Fixes #4756

### CODE REVIEW

File: devdocs/process.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      If you did not get a review within 2-3 days, it is OK to request for a review 
      by posting a comment in the PR. 
 
-   * Once the PR is open, try and complete it within 2 weeks. Inactivity for a longer period would necessitate a restart of the PR.
+   * Once the PR is open, try and complete it within 2 weeks. Inactivity for a 
+     longer period would necessitate a restart of the PR.
```
Comment (thyageshm) [inline devdocs/process.md]: sir I dont think this makes any difference to the output file. Did you intend this modification?

File: devdocs/process.md [hunk 2]
```diff
@@ -271,6 +276,13 @@ The meaning of label group prefixes:
   Not to be applied for issues with a priority `p.high` or above.
 * `d.Committers`: More difficult issues that are better left for committers or more senior developers.
 
+**Effort**
+
+This label is used to indicate how much effort is expected for (or was spent on)
+an issue.
+
+`e.1` is roughly equal to an hour of work, `e.2` is two hours of work, and so on.
```
Comment (thyageshm) [inline devdocs/process.md]: If the effort value is going to be a running number, how many such labels would we have? 
To address this, shall we say that if the expected effort exceeds the max effort label, we should break it down into smaller issues

File: devdocs/process.md (hunk 1 above)
Comment (damithc) [inline devdocs/process.md]: Added a line break. The original line was longer than 110 chars.

File: devdocs/process.md (hunk 2 above)
Comment (damithc) [inline devdocs/process.md]: We have 1,2,4,8,16 only. This sequence is often used for evaluating the size of a user story.

File: devdocs/process.md (hunk 1 above)
Comment (thyageshm) [inline devdocs/process.md]: oh ok! ooops sorry about that...

File: devdocs/process.md (hunk 2 above)
Comment (thyageshm) [inline devdocs/process.md]: I see. Should we add the point that the issue should be broken up if it exceeds such a number? Also would we want to have `e.16`?

File: devdocs/process.md [hunk 3]
```diff
@@ -205,7 +207,10 @@ RL:
   * Update milestone.
     * State the release number in the milestone notes.
     * Ensure all issues and PRs included in the release are tagged with the correct milestone.
+    * For issues in the milestone: assign/revise `e.` labels  based on the estimated
```
Comment (thyageshm) [inline devdocs/process.md]: I think there is a double space between "labels" and "based"

**Comment by thyageshm** [comment]
I have no more comments :-)

File: devdocs/process.md (hunk 2 above)
Comment (damithc) [inline devdocs/process.md]: Yes, I'll mention smaller the better, but I'm planning to do that in a separate document (i.e. common guidelines for all our projects). We'll keep `e.16` for rare cases requiring about 2 days of work.

**Pull request merged**